lighthouse-core/audits/time-to-first-byte.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit');
+const URL = require('../lib/url-shim');
+const Util = require('../report/v2/renderer/util');
+
+const TTFB_THRESHOLD = 600;
+const TTFB_THRESHOLD_BUFFER = 15;
+
+class TTFBMetric extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Performance',
+      name: 'time-to-first-byte',
+      description: 'Time To First Byte (TTFB)',
```

Comment:
this description nicely summarizes what we're really doing but it feels weird along with the other perf opportunities, maybe we should convert it into something like "Keep server response times low" and mention TTFB in the description or debugString?

I know @paulirish doesn't like "response time" though ;) open to whatever doesn't make it feel out of place there.

Reply:
Keep server response times low (TTFB)?